### PR TITLE
Update Unweighted_Shortest_Paths.mdx

### PR DESCRIPTION
**Pull request by TheScarletPhoenix**
added python solution (tested code with proof of ac)

_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [X] I have tested my code.
- [X] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [X] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [X] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by TheScarletPhoenix** [comment]
image

File: content/4_Gold/Unweighted_Shortest_Paths.mdx [hunk 1]
```diff
@@ -248,6 +248,65 @@ public class Solution {
 
 </JavaSection>
 
+<PySection>
+
+```py
+from collections import deque, defaultdict
+
+
+def find_shortest_path(n, m, edges):
```
Comment (ryanchou-dev) [inline content/4_Gold/Unweighted_Shortest_Paths.mdx:257]: does this need to be in a function?

Comment (TheScarletPhoenix) [reply content/4_Gold/Unweighted_Shortest_Paths.mdx]: no, it still works, so i updated the code.

image

File: content/4_Gold/Unweighted_Shortest_Paths.mdx [hunk 2]
```diff
@@ -248,6 +248,58 @@ public class Solution {
 
 </JavaSection>
 
+<PySection>
+
+```py
+from collections import deque, defaultdict
+
+# Reading input
```
Comment (ryanchou-dev) [inline content/4_Gold/Unweighted_Shortest_Paths.mdx:256]:
```suggestion
```

File: content/4_Gold/Unweighted_Shortest_Paths.mdx [hunk 3]
```diff
@@ -248,6 +248,58 @@ public class Solution {
...
+n, m = map(int, input().split())
+edges = []
+for _ in range(m):
+	a, b = map(int, input().split())
+	edges.append((a, b))
+
+# Constructing the graph
```
Comment (ryanchou-dev) [inline content/4_Gold/Unweighted_Shortest_Paths.mdx:263]:
```suggestion
```

File: content/4_Gold/Unweighted_Shortest_Paths.mdx [hunk 4]
```diff
@@ -248,6 +248,58 @@ public class Solution {
...
+			if neighbor == n:  # If we reach the destination
+				# Reconstruct the path
+				path = []
+				current = n
+				while current != -1:
+					path.append(current)
+					current = parent[current]
+				path.reverse()
+
+				# Outputting the result
```
Comment (ryanchou-dev) [inline content/4_Gold/Unweighted_Shortest_Paths.mdx:292]:
```suggestion
```

**Review by ryanchou-dev — APPROVED** [review]
nice!

**Pull request merged**